cypress/integration/headings_spec.js

```diff
@@ -29,4 +29,13 @@ describe("<headings>", () => {
           .should('eq', "ERROR (E0005): Headings should not skip levels.")
       });
   });
+
+  it('should not have role=text', () => {

```

Comment:
Thank you for adding this! I've been meaning to get around to refactoring these tests. _(No action needed)_